jaxrs-api/src/test/java/jakarta/ws/rs/core/CookieTest.java

```diff
@@ -46,12 +46,12 @@ public void tearDown() throws Exception {
     public void testEquals() {
         System.out.println("equals");
         Object nullObj = null;
-        Cookie cookie = new Cookie("name", "value");
```

Comment:
As we must guarantee backwards compatibility, we must not remove these tests.

Reply:
You are right I should have left existing tests unchanged and add new ones.